### ISSUE TITLE
Tensorflow.js looks for CUDA 10.0 with libtensorflow recompiled against CUDA 10.1 

### ISSUE THREAD

**Issue by sbrl** [tensorflow/tfjs#3490]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
2.0.0

#### Browser version
n/a, Node.js v14.2.0

#### Describe the problem or feature request
If I recompile libtensorflow against CUDA 10.1.168 according to the instructions [here](https://github.com/tensorflow/tfjs/blob/master/tfjs-node/README.md#optional-build-optimal-tensorflow-from-source) and substitute it in as instructed into my Tensorflow.js install, it is observed that Tensorflow.js continues to fail to load the CUDA 10.0 binaries.

For the exact steps I followed to compile libtensorflow, see [this comment I made on a different issue](https://github.com/tensorflow/tensorflow/issues/40614#issuecomment-647690028). Following that, I did the following:

```bash
cp bazel-bin/tensorflow/tools/lib_package/libtensorflow.tar.gz ~/myproject/node_modules/@tensorflow/tfjs-node/deps
cp bazel-bin/tensorflow/tools/lib_package/libtensorflow.tar.gz ~/myproject/node_modules/@tensorflow/tfjs-node-gpu/deps
cd ~/path-to-my-project/node_modules/@tensorflow/tfjs-node/deps
tar -xf libtensorflow.tar.gz
cd ~/path-to-my-project/node_modules/@tensorflow/tfjs-node-gpu/deps
tar -xf libtensorflow.tar.gz
```

...as instructed by your README (link above).

Specifically, I'm getting this error: https://ybin.me/p/2007f2550ecf75a4#m91xA9jifKJumfGEs1YOpANyeuRKrtTI4m7SnfgdM6I=

(it's a lot of output, so I used a pastebin).

To be clear, I recompiled libtensorflow against CUDA **10.1**, but even after substituting it in Tensorflow.js complains it can't find CUDA **10.0**.

#### Code to reproduce the bug / link to feature request
Any example Tensorflow.js program for Node.js (it doesn't even have to do anything).

**Comment by pyu10055** [comment]
@sbrl The error seems to indicate TensorFlow is looking for CUDA 10.0, can you check if you are compiling TF 2.x? tfjs-node should be using TF C version [1.15.0](https://github.com/tensorflow/tfjs/blob/master/tfjs-node/scripts/deps-constants.js#L25), it is likely tfjs-node-gpu still using the 1.15 so, and it failed to find 10.0 CUDA.

You might have to hack the line if you are compiling 2.x, and have tfjs-node pointing at the right so file.

**Comment by sbrl** [comment]
Hey  there,  @pyu10055! GH seems to have deleted my comment from earlier - that's strange :thinking: 

Anyway, I have tried building Tensorflow 1.15.0, but I can't continue because I can't find a version of bazel that it accepts. If I use the 2.0.0 version from previously, it claims it needs bazel 0.62.1 or lower. If I download  v0.26.1, then it claims that I've actually downloaded v0.19.2, and that I need upgrade..... so I'm confused.
